org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -1523,7 +1523,9 @@ class CompletionItem {
 
 	/**
 	 * Indicates if this item is deprecated.
+	 * @deprecated Use `tags` instead if supported.
```

Comment:
Separate with a new line to be consistent with other `@deprecated` Javadoc tags.